h/accounts/test/views_test.py

```diff
@@ -349,7 +349,7 @@ def test_forgot_password_creates_no_activations_when_validation_fails(activation
 @patch('h.accounts.views.reset_password_link')
 @forgot_password_fixtures
 def test_forgot_password_generates_reset_link(reset_link, authn_policy):
-    request = DummyRequest(method='POST')
```

Comment:
You still don't need the additional `authenticated_userid` keyword. Or at least you shouldn't. As mentioned above, that's what the `authn_policy` fixture is supposed to be doing.